app/Http/Controllers/PayPalController.php

```diff
@@ -1,54 +1,32 @@
 <?php
 namespace App\Http\Controllers;
 use Illuminate\Http\Request;
-use Laravel\Cashier\Subscription;
-use Carbon\Carbon;
+use App\Services\PayPalService;
 use Srmklive\PayPal\Services\PayPal as PayPalClient;
 
 class PayPalController extends Controller
 {
+
+    /** @var PaypalService */
+    protected $service;
+
+    /**
+     * AbilityController constructor.
+     * @param PayPalService $service
+     */
+    public function __construct(PayPalService $service)
+    {
+        $this->service = $service;
```

Comment:
Missing the middlewares to check that the user is logged in and not impersonating someone else